### DISCUSSION TITLE
February update

### DISCUSSION THREAD

**Discussion by marcusolsson** [marcusolsson/obsidian-projects#402] [Announcements]
The Projects plugin is now on version 1.11.5, and it's the most stable and feature-rich version so far! 🎉 

I'm really happy with where the plugin is going so far. A lot has happened since my last post, so I wanted to highlight some of the improvements since.

- **Field configuration**—settings that apply to a given field across notes. So far, you can configure predefined values for text fields, to make them show up in board view, even if no notes uses that field. Text fields now support Markdown formatting by enabling rich text formatting for the text fields that contains Markdown syntax.

- **Include fields** to Board and Gallery view lets you add any field as metadata in the card. Many notes don't have a very readable name, so being able to add information from other fields really helps getting an overview and finding the right cards.

- **Tag data source**—many of you don't use folder to organize your notes. With the new tag data source, you can instead choose the notes to include by a tag instead.

- We've also released a ton of bug fixes. I'm particularly excited about finding a fix for a frustrating bug where the view would rerender any time you changed it's config. Being able to resize columns and hide multiple fields without the view resetting makes it so much more enjoyable to configure your views.

In more personal news, I'm marrying my best friend next month 🎉 I'll be away from the keyboard for a while, so don't expect any major releases for the weeks to come. I'll also likely be slow to respond to comments and new issues. In the meantime, check out [CONTRIBUTING](https://github.com/marcusolsson/obsidian-projects/blob/main/CONTRIBUTING.md) if you'd like to help move the project forward while I'm away.

Thanks for using Projects! ❤️

**Comment by Tobi-De** [comment]
Thanks for making this and congratulations for your wedding.
